test_app/tests/conftest.py

```diff
@@ -179,6 +179,33 @@ def github_enterprise_team_authenticator(github_enterprise_team_configuration):
     authenticator.delete()
 
 
+@pytest.fixture
+def oidc_configuration():
+    return {
+        "OIDC_ENDPOINT": "https://localhost/api/gateway/callback/github_test/",
```

Comment:
NIT: Looks like there is a github reference here can we change it to `oidc_test`?